po/sr@latin.po

```diff
@@ -142,246 +146,250 @@ msgid "Cycle through windows"
 
-#: Settings.ui.h:48
+#: Settings.ui.h:51
 msgid "Behaviour when clicking on the icon of a running application."
 msgstr "Ponašanje pri kliku na pokrenuti program."
 
-#: Settings.ui.h:49
+#: Settings.ui.h:52
 msgid "Click action"
 msgstr "Radnja klika"
 
-#: Settings.ui.h:50
+#: Settings.ui.h:53
 msgid "Minimize"
 msgstr "minimizuj"
 
-#: Settings.ui.h:51
-#, fuzzy
-msgid "Minimize or overview"
-msgstr "minimiuj prozor"
-
-#: Settings.ui.h:52
-#, fuzzy
+#: Settings.ui.h:54
 msgid "Behaviour when scrolling on the icon of an application."
-msgstr "Ponašanje pri kliku na pokrenuti program."
+msgstr "Ponašanje pri klizanju na ikonu pokrenutog programa."
 
-#: Settings.ui.h:53
-#, fuzzy
+#: Settings.ui.h:55
 msgid "Scroll action"
-msgstr "Radnja klika"
+msgstr "Radnja kklizanja"
 
-#: Settings.ui.h:54
+#: Settings.ui.h:56
 msgid "Do nothing"
 msgstr "ništa"
 
-#: Settings.ui.h:55
+#: Settings.ui.h:57
 msgid "Switch workspace"
-msgstr ""
+msgstr "Prebaci radni prostor"
 
-#: Settings.ui.h:56
+#: Settings.ui.h:58
 msgid "Behavior"
 msgstr "Ponašanje"
 
-#: Settings.ui.h:57
-#, fuzzy
+#: Settings.ui.h:59
 msgid ""
 "Few customizations meant to integrate the dock with the default GNOME theme. "
 "Alternatively, specific options can be enabled below."
 msgstr ""
-"Nekoliko postavki je namenjeno ugrađivanju doka u osnovnu temu Gnoma. "
+"Nekoliko postavki namenjenih ugrađivanju doka u osnovnu temu Gnoma. "
 "Alternativno, posebne postavke se mogu urediti ispod."
 
-#: Settings.ui.h:58
+#: Settings.ui.h:60
 msgid "Use built-in theme"
 msgstr "Koristi ugrađenu temu"
 
-#: Settings.ui.h:59
+#: Settings.ui.h:61
 msgid "Save space reducing padding and border radius."
 msgstr "Čuva prostor sužavanjem popune i opsega ivica."
 
-#: Settings.ui.h:60
+#: Settings.ui.h:62
 msgid "Shrink the dash"
 msgstr "Skupi ploču"
 
-#: Settings.ui.h:61
+#: Settings.ui.h:63
 msgid "Show a dot for each windows of the application."
 msgstr "Prikazuje tačku za svaki prozor programa."
 
-#: Settings.ui.h:62
+#: Settings.ui.h:64
 msgid "Show windows counter indicators"
 msgstr "Prikaz indikatora brojača prozora."
 
-#: Settings.ui.h:63
+#: Settings.ui.h:65
 msgid "Set the background color for the dash."
-msgstr ""
+msgstr "Postavi pozadinsku boju ploče."
 
-#: Settings.ui.h:64
+#: Settings.ui.h:66
 msgid "Customize the dash color"
-msgstr ""
+msgstr "Prilagodi boju ploče"
 
-#: Settings.ui.h:65
+#: Settings.ui.h:67
 msgid "Tune the dash background opacity."
 msgstr "Prilagodi prozirnost pozadine ploče."
 
-#: Settings.ui.h:66
+#: Settings.ui.h:68
 msgid "Customize opacity"
 msgstr "Prilagoćavanje prozirnosti"
 
-#: Settings.ui.h:67
+#: Settings.ui.h:69
 msgid "Opacity"
 msgstr "Prozirnost"
 
-#: Settings.ui.h:68
+#: Settings.ui.h:70
+msgid "Enable Unity7 like glossy backlit items"
+msgstr "Uključi šljašteće pozadinske stavke poput Unity7"
+
+#: Settings.ui.h:71
 msgid "Force straight corner\n"
-msgstr ""
+msgstr "Primoraj ravne uglove\n"
```

Comment:
Nateraj ravne uglove

Reply:
Thank you @cra0zy  for translation review and suggestions. I'll be fixing errors and sending changes shortly. I replied on individual term suggestions above.